compiler/packagehandling.nim

```diff
@@ -39,11 +39,11 @@ proc getPackageName*(conf: ConfigRef; path: string): string =
     if parents <= 0: break
 
 proc fakePackageName*(conf: ConfigRef; path: AbsoluteFile): string =
-  # foo/../bar becomes foo@..@bar
-  result = relativeTo(path, conf.projectPath, '/').string.multiReplace({"/": "@", "@": "@@"})
+  # foo-#head/../bar becomes @foo-@shead@..@bar
```

Comment:
Is this example wrong or am I misunderstanding what is happening? (probably it is the latter, but I want to double-check)

If I'm reading replacing correctly, `#head` should become `@hhead`, not `@shead`?
Also, shouldn't `/../`become `@s..@s`, and not `@..@`?

Reply:
My bad, fixed.